user/pages/03.Howtos/10.l3-high-availability/docs.en.md

```diff
@@ -1,11 +1,12 @@
 ---
 title: 'Building Layer 3 High Availability'
 published: true
-date: '16-09-2018 16:20'
+date: '29-03-2024 10:20'
 taxonomy:
     category:
         - docs
 ---
+! This solution is deprecated, we highly recommend to use Octavia LBaaS for this purpose: [Create a Load Balancer](../../02.Tutorials/05.lbaas/docs.en.md)
```

Comment:
This approach is not deprecated, it is perfectly valid for certain use cases but can be tedious compared to LBaaS, so using Octavia LBaaS should at least be considered before opting for this variant.

Reply:
Will the following sentence be more accurate?

"Please consider using Octavia LBaaS solution provided by the SysEleven Stack before you decide to create a VIP VRRP cluster on your own."

Reply:
Yes, and let's link "Octavia LBaaS" to the appropriate page in our documentation.

Reply:
Yes, and also link to the according page.